components/Modal.js

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import { StyleSheet, Text, View, Dimensions, Image, TouchableOpacity } from 'react-native';
+
+
+export default class App extends React.Component {
```

Comment:
This is a mistake that you shouldn't be making anymore! 😫- make sure that your classes match your file names.

Also, be more specific with your file names. `Modal` insinuates that it's general and holds multiple purposes. However, this file should be named `StatusUpdateModal` since you are updating statuses. Think about what the title implies!